keras/backend/common.py

```diff
@@ -192,6 +191,9 @@ def set_image_dim_ordering(dim_ordering):
         >>> K.image_data_format()
         'channels_last'
     ```
+
+    # Raises
+        ValueError if invalid `'dim_ordering'`
```

Comment:
Since `dim_ordering` is an argument name and not a string variable, quotes are not necessary.